cirq-google/cirq_google/workflow/quantum_runtime.py

```diff
@@ -80,10 +81,12 @@ class RuntimeInfo:
             `cg.QuantumExecutable` was executed.
         qubit_placement: If a QubitPlacer was used, a record of the mapping
             from problem-qubits to device-qubits.
+        timings: A dictionary of the duration of various steps in the workflow.
```

Comment:
Be more specific.  What are the keys and values?  Are the values in seconds, millis, micros, nanos, etc?
Also, I would consider changing the name to include units for clarity.

Reply:
added docstrings and changed name